contracts/core/libraries/Transfer.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.19;
 
 // External Imports
-import {SafeTransferLib} from "solady/utils/SafeTransferLib.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
curious . why are we switching ?

Reply:
solady's SafeTransferLib doesnt check for tokens code.length to check if the token is a contract. Also the library does not handle incorrect or unexpected return data from token contracts and some transactions may pass silently, even if the underlying transfer has failed or is not executed correctly.